Update yup peer dependency

The yup related packages are currently still relying on yup 0.32.9. But in the last 3 months 1.0.0 has been released.

https://www.npmjs.com/package/yup?activeTab=versions

```
npm ERR! Conflicting peer dependency: yup@0.32.11
npm ERR! node_modules/yup
npm ERR!   peer yup@"^0.32.9" from @felte/validator-yup@1.0.11
npm ERR!   node_modules/@felte/validator-yup
npm ERR!     dev @felte/validator-yup@"^1.0.11" from the root project
```

--- a/.changeset/tame-snakes-return.md
+++ b/.changeset/tame-snakes-return.md
@@ -1,0 +1,5 @@
+---
+'@felte/validator-yup': minor
+---
+
+Update yup peer dependency to 1.0

--- a/packages/validator-yup/package.json
+++ b/packages/validator-yup/package.json
@@ -44,10 +44,10 @@
     "@felte/core": "workspace:*",
     "felte": "workspace:*",
     "svelte": "^3.46.4",
-    "yup": "^0.32.9"
+    "yup": "^1.2.0"
   },
   "peerDependencies": {
-    "yup": "^0.32.9"
+    "yup": ">=1.2.0"
   },
   "publishConfig": {
     "access": "public"

